content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -37,7 +37,7 @@ job:
     variable when submitting the training job.
 
     ```
-    $ STUDIO_TOKEN=**** dvc exp run
+    $ export STUDIO_TOKEN=**** dvc exp run
```

Comment:
If we want to use `export`, it would have to be

```suggestion
    $ export STUDIO_TOKEN=**** 
    $ dvc exp run
```

Reply:
@jendefig It's not mandatory to use `export` if the user is only trying to pass the value of the env var this one time. It's ok to use `export`, but is there any particular reason you're making this change?

Reply:
p.s. should prob use `cli` highlighter on that block tho.

Reply:
Best to keep a single line if no other criteria to pick is clear.

Reply:
We could leave as is (no `export`) and put an admon underneath like:

```
In this example, the `STUDIO_TOKEN` environment variable is passed in the same line as `dvc exp run` so that it only applies to that line. If you want all experiments in this environment to be sent to Studio, use `export STUDIO_TOKEN=***`.
```